CMakeLists.txt

```diff
@@ -0,0 +1,428 @@
+cmake_minimum_required(VERSION 3.15)
+
+# Absolute path to FreeRTOS config file directory
+set(FREERTOS_CONFIG_FILE_DIRECTORY "" CACHE STRING "Absolute path to the directory with FreeRTOSConfig.h")
```

Comment:
We should make this into a command line argument. And if it is not specified - issue an error:
```
if( NOT FREERTOS_CONFIG_FILE_DIRECTORY )
    message( ERROR "FreeRTOSConfig.h file directory not specified. Please specify it using -DFREERTOS_CONFIG_FILE_DIRECTORY='/path/to/your/file' while running CMake" ) 
endif()
```

Reply:
Fixed.